### PR TITLE
ORLP-413 Change spagetti code to normal view in controllers

### PR DESCRIPTION
**Pull request by romandr**

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/controller/UserCardQueueController.java [hunk 1]
```diff
@@ -22,23 +22,26 @@
...
             throws NotAuthorisedUserException, IllegalArgumentException {
         userCardQueueService.updateUserCardQueue(deckId, cardId, status);
         return ResponseEntity.ok().build();
     }
 
     @GetMapping("/api/private/decks/{deckId}/cards-that-need-repeating/count")
-    public ResponseEntity<Long> countCardsThatNeedRepeating(@PathVariable Long deckId) throws
-            NotAuthorisedUserException {
+    public ResponseEntity<Long> countCardsThatNeedRepeating(
+            @PathVariable Long deckId) throws NotAuthorisedUserException {
```
Comment (MoGUREV) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/UserCardQueueController.java]: Return to inital view, but move throws declaration to new line

File: src/main/java/com/softserve/academy/spaced/repetition/security/controller/AuthenticationRestController.java [hunk 2]
```diff
@@ -28,21 +28,25 @@
...
+                                                                                         Device device) {
         HttpHeaders headers = authenticationRestService.getGoogleHeaders(idToken, device);
         return new ResponseEntity<>(new JwtAuthenticationResponse("Ok"), headers, HttpStatus.OK);
     }
 
     @Auditable(action = AuditingAction.SIGN_IN_VIA_FACEBOOK)
     @RequestMapping(value = "${app.social.facebook.path}", method = RequestMethod.POST)
-    public ResponseEntity<JwtAuthenticationResponse> createAuthenticationTokenFromFacebook(@RequestBody String token, Device device) throws GeneralSecurityException, IOException {
+    public ResponseEntity<JwtAuthenticationResponse> createAuthenticationTokenFromFacebook(
+            @RequestBody String token, Device device) throws GeneralSecurityException, IOException {
```
Comment (MoGUREV) [inline src/main/java/com/softserve/academy/spaced/repetition/security/controller/AuthenticationRestController.java]: Try to move `@RequestBody String token` on previous lane

File: src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java [hunk 3]
```diff
@@ -35,52 +35,68 @@
...
-    public ResponseEntity<List<CommentDTO>> getAllCommentsByCourse(@PathVariable Long categoryId, @PathVariable Long courseId) {
+    public ResponseEntity<List<CommentDTO>> getAllCommentsByCourse(@PathVariable Long categoryId,
+                                                                   @PathVariable Long courseId) {
         LOGGER.debug("View all comments for course with id: {}", courseId);
         List<Comment> courseCommentsList = courseCommentService.getAllCommentsForCourse(courseId);
-        Link collectionLink = linkTo(methodOn(CourseCommentController.class).getAllCommentsByCourse(categoryId, courseId)).withSelfRel();
-        List<CommentDTO> listOfComments = DTOBuilder.buildDtoListForCollection(courseCommentsList, CommentDTO.class, collectionLink);
+        Link collectionLink = linkTo(methodOn(CourseCommentController.class)
+                .getAllCommentsByCourse(categoryId, courseId)).withSelfRel();
+        List<CommentDTO> listOfComments = DTOBuilder.buildDtoListForCollection(courseCommentsList,
```
Comment (VovaChornyi) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/CourseCommentController.java]: make next line after "." not "," to better code view

File: src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java [hunk 4]
```diff
@@ -28,11 +28,16 @@
...
 
-    private static final EnumMap<AccountStatus, ResponseEntity<MessageDTO>> USER_STATUS_ERROR_RESPONSE = new EnumMap<>(AccountStatus.class);
+    private static final EnumMap<AccountStatus,
+            ResponseEntity<MessageDTO>> USER_STATUS_ERROR_RESPONSE = new EnumMap<>(AccountStatus.class);
 
     static {
-        USER_STATUS_ERROR_RESPONSE.put(AccountStatus.DELETED, new ResponseEntity<>(new MessageDTO("Account with this email is deleted"), HttpStatus.LOCKED));
-        USER_STATUS_ERROR_RESPONSE.put(AccountStatus.BLOCKED, new ResponseEntity<>(new MessageDTO("Account with this email is blocked"), HttpStatus.FORBIDDEN));
+        USER_STATUS_ERROR_RESPONSE
+                .put(AccountStatus.DELETED, new ResponseEntity<>(new MessageDTO("Account with this email is deleted"),
```
Comment (VovaChornyi) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java]: call method in this line and make 3 line for better code view

File: src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java [hunk 5]
```diff
@@ -28,11 +28,16 @@
...
+                        HttpStatus.LOCKED));
+        USER_STATUS_ERROR_RESPONSE
```
Comment (VovaChornyi) [inline src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java]: same to line 35

**Pull request merged**